modules/example/src/main/scala/zio/telemetry/example/config/Configuration.scala

```diff
@@ -0,0 +1,25 @@
+package zio.telemetry.example.config
+
+import pureconfig.ConfigSource
+import pureconfig.generic.auto._
+import zio.{ RIO, Task }
+
+trait Configuration extends Serializable {
+  val config: Configuration.Service[Any]
+}
+
+object Configuration {
+
+  trait Service[R] {
+    val load: RIO[R, Config]
+  }
+
+  trait Live extends Configuration {
+    override val config: Service[Any] = new Service[Any] {
+      val load: Task[Config] = Task.effect(ConfigSource.default.loadOrThrow[Config])
+    }
+  }
+
+  object Live extends Live
```

Comment:
Drop it if you don't use it.

Reply:
It's used in [ProxyServer](https://github.com/zio/zio-telemetry/pull/19/files#diff-777a26bc95f4a2a5e38d61f3464ce55bR17).